libs/agno/agno/eval/accuracy.py

```diff
@@ -162,6 +164,8 @@ class AccuracyEval:
     print_results: bool = False
     # Save the result to a file
     save_result_to_file: Optional[str] = None
+    # Track the results, for them to be available in the Agno Platform
+    track_results: bool = False
```

Comment:
Lets have the same as agents and teams and make it `monitor`. Also make it toggle-able with the environment variable (check `agent.py`)

Reply:
Good one